src/pybind/mgr/dashboard/frontend/cypress/integration/cluster/hosts.po.ts

```diff
@@ -182,7 +184,7 @@ export class HostsPageHelper extends PageHelper {
 
     this.clickTab('cd-host-details', hostname, 'Daemons');
     cy.get('cd-host-details').within(() => {
-      cy.wait(10000);
+      cy.wait(20000);
```

Comment:
Every time we increase timeouts we should explain why that much (e.g.: this doubled the time because "x") and evaluate if something can be sped up by parallelization/threading/etc.

In general, we should take high timeouts in tests as a smell of poor responsivity or scalability issues in our code (are we perhaps fetching too many unnecessary data in the background?).

Reply:
Sorry, I forgot to add justification here. But I saw that the time taken to drain a host depends on the services present inside that hosts. If the host has any OSDs present, then it takes a while to finish that check. Because of this, our tests were failing (not everytime, but 2 or 3 out of 5 runs). that's why I increased the timeout.